lib/salus/config.rb

```diff
@@ -22,14 +22,22 @@ class Config
     attr_accessor :active_scanners
 
     # Dynamically get all Scanner classes
-    ABSTRACT_SCANNERS = %i[Base NodeAudit].freeze
+    ABSTRACT_SCANNERS = %i[Base NodeAudit LanguageVersion].freeze
     SCANNERS = Salus::Scanners.constants
       .reject { |klass| ABSTRACT_SCANNERS.include?(klass) }
       .map { |klass| [klass.to_s, Salus::Scanners.const_get(klass)] }
       .sort
```

Comment:
For readability (of const being set once) what do you think about adding this to the existing SCANNERS =
 
```
.merge(Salus::Scanners::LanguageVersion.constants
.reject { |klass| ABSTRACT_SCANNERS.include?(klass) }
.map { |klass| [klass.to_s, Salus::Scanners::LanguageVersion.const_get(klass)] }
```
before the .sort .to_h .freeze?